vellum/workflows/triage_message/nodes/schedule_meeting_node.py

```diff
@@ -0,0 +1,38 @@
+from vellum.workflows.nodes import BaseNode
+from .read_message_node import ReadMessageNode
+from .parse_function_call_node import ParseFunctionCallNode
+
+
+class ScheduleMeetingNode(BaseNode):
+    scheduling_link = ParseFunctionCallNode.Outputs.parameters["scheduling_link"]
+    contact_email = ReadMessageNode.Outputs.message["contact_email"]
+    contact_full_name = ReadMessageNode.Outputs.message["contact_full_name"]
+    contact_phone_number = ReadMessageNode.Outputs.message["contact_phone_number"]
+    
+    class Outputs(BaseNode.Outputs):
+        summary: str
+    
+    def run(self) -> Outputs:
+        
+        provider = self._detect_provider(self.scheduling_link)
+        
+        summary = f"Successfully detected {provider} scheduling link: {self.scheduling_link}"
+        
+        if self.contact_email:
+            summary += f" for contact {self.contact_email}"
+        
+        return self.Outputs(summary=summary)
```

Comment:
Introduce State in this workflow with a `action_history` array. Append to that array with the args and results of this node, then feed that state as a prompt input to triage message node